arangod/Aql/Function.h

```diff
@@ -50,36 +54,50 @@ struct Function {
     /// cache
     Cacheable = 2,
 
-    /// @brief whether or not the function may be executed on DB servers
-    CanRunOnDBServer = 4,
+    /// @brief whether or not the function may be executed on DB servers, 
+    /// general cluster case (non-OneShard)
+    CanRunOnDBServerCluster = 4,
```

Comment:
Are there actually (or potentially) any functions that can be executed on a non-OneShard cluster server, but not on a OneShard server? To me it feels like `CanRunOnDBServerCluster` should imply `CanRunOnDBServerOneShard`.

Reply:
I added more comments to explain the rationale behind it.